### DISCUSSION TITLE
Delete files not needed?

### DISCUSSION THREAD

**Discussion by evie3211** [MiczFlor/RPi-Jukebox-RFID#1587] [Q&A]
I'm fairly new to coding so please be patient with my ignorance. I don't see a way to delete folders on the web app. And, if I downloaded them from YouTube I don't see a way to delete them.

For example, I uploaded a file and the name contained an apostrophe, which the system did not like referencing. So I would like to delete it.

In the second photo, I want to delete the YouTube downloaded content "Danny Go" because it didn't download everything I wanted and I created a new file with all the content I wanted in it.

I know I can make new files and just not have them linked to any RFID chip, just wanting to try to keep my SD card tidy even when I make mistakes on learning the system. Thanks!

<img width="1131" alt="Screen Shot 2021-10-22 at 12 23 38 AM" src="https://user-images.githubusercontent.com/92416517/138397699-c4546061-1d83-490b-9ec2-239e62800913.png">
<img width="1131" alt="Screen Shot 2021-10-22 at 12 24 56 AM" src="https://user-images.githubusercontent.com/92416517/138397818-7004fb5a-f5aa-47c4-942f-ea2d682e6eee.png">

**Comment by Thilo-90** [comment]
Hi @evie3211 
i am quite new to the Phoniebox and coding stuff too.
I did also not find a delete option in the webapp. Therefor i used the console vis ssh connection to clean everything up.

It is quite easy. Navigate to the shared audio folder
`cd RPi-Jukebox-RFID/shared/audiofolders/`

Look up folder contents
`ls`

and delete the folders you do not wish to keep.
`rm –Rf foldername`

for suggest reading [this](https://www.dummies.com/computers/raspberry-pi/remove-directories-raspberry-pi/) for further info.

**Reply by evie3211** [reply]
Thank you! I’m also new to coding but that makes sense! I’ll definitely try it.
